client/src/components/Register.js

```diff
@@ -77,7 +83,16 @@ class Register extends React.Component {
             I am a volunteer
           </label>
           <label className="terms">By clicking Register, you agree to the site
-            <Link to="/terms"> terms</Link>
+            <Link onClick={this.handleOpen}> terms</Link>
+            <Modal
+              isOpen={this.state.open}
+              onRequestClose={this.handleClose}
+              contentLabel="Example Modal"
+            >
+              <h2>Terms</h2>
```

Comment:
It might be nice to pull this out into it's own component at some point. ...Note to future self